spec/src/main/asciidoc/changelog.asciidoc

```diff
@@ -21,12 +21,17 @@
 
 === Release 2.0
 
+==== Incompatible changes
+
+* Update OpenTracing API to 0.33.0 (https://github.com/eclipse/microprofile-opentracing/pull/177[#177])
+* Use Jakarta EE 8 APIs instead of Java EE 7 and remove dependency on Jackson (https://github.com/eclipse/microprofile-opentracing/pull/162[#162])
```

Comment:
Use Jakarta EE 8 APIs instead of Java EE7 should not be an incompatible changes as the package names are the same.

Reply:
My understanding is that the bump to Jakarta EE API was moved to a breaking change release bc there were some incompatible changes cc) @kenfinnigan

Reply:
For some MP specs the move to Jakarta EE 8 APIs was also a breaking change because they weren't marked as "provided".

For Open Tracing I don't think that was the case, so the update wouldn't be a breaking change.

Reply:
ack I have moved the jakarta to other changes section.

@Emily-Jiang I have added more details regarding the breaking changes and migration.